docs/components/console/manage-clusters/update-cluster.md

```diff
@@ -0,0 +1,21 @@
+---
+id: update-cluster
+title: Update your cluster
+description: "Clusters eligible for updates will show a button on the UI."
+---
+
+:::note
+This action cannot be undone. Updated clusters cannot be reverted to the previous version.
+:::
+
+Clusters can be updated to new versions of Camunda 8 manually or automatically.
+
+At this time, updates do not trigger backups.
```

Comment:
Maybe we can add that you can create backup manually before updating if you want

Reply:
@chaima-mnsr is this available today for SaaS through the console or only the cluster as documented [here](https://docs.camunda.io/docs/components/concepts/backups/)?

Reply:
yeah it is available through console since camunda 8.2 
<img width="866" alt="image" src="https://github.com/camunda/camunda-docs/assets/25367434/2d58f96f-92e8-4529-ad34-d5cd7f0e511e">

Reply:
@chaima-mnsr, I will open an issue for this topic. I was expecting to see some documentation on how to create a backup in SaaS, but I only see conceptual content that mentions [programmatic access](https://docs.camunda.io/docs/components/concepts/backups/#programmatic-access).

This section may not be valid either - https://docs.camunda.io/docs/components/concepts/backups/#restore